apps/galleries_institutions/components/filter_facet/locationSynonyms.coffee

```diff
@@ -0,0 +1,4 @@
+module.exports = [
+  ['st', 'saint']
```

Comment:
I think that would work if we had one known target string and one known replacement string, but the idea here is to accommodate arbitrary-length lists of synonyms which should _all_ be treated as equivalent.

(My examples may be misleading since each has only a pair of synonyms, but imagine instead we wanted ["saint", "st", "san", "santo", "santa"] to all be treated as synonyms of one another.)